packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -418,171 +419,205 @@ const useQuickActions = () => {
     [handleMouseDown, isRTL]
   );
 
-  const noElementSelectedActions = useMemo(() => {
-    return [
-      {
-        Icon: Bucket,
-        label: ACTIONS.CHANGE_BACKGROUND_COLOR.text,
-        onClick: (evt) => {
-          handleFocusPageBackground(backgroundElement?.id)(evt);
-
-          trackEvent('quick_action', {
-            name: ACTIONS.CHANGE_BACKGROUND_COLOR.trackingEventName,
-            element: 'none',
-          });
-        },
-        ...actionMenuProps,
-      },
-      {
-        Icon: Media,
-        label: ACTIONS.INSERT_BACKGROUND_MEDIA.text,
-        onClick: (evt) => {
-          handleFocusMediaPanel()(evt);
-
-          trackEvent('quick_action', {
-            name: ACTIONS.INSERT_BACKGROUND_MEDIA.trackingEventName,
-            element: 'none',
-          });
-        },
-        separator: 'top',
-        ...actionMenuProps,
-      },
-      {
-        Icon: LetterTPlus,
-        label: ACTIONS.INSERT_TEXT.text,
-        onClick: (evt) => {
-          handleFocusTextSetsPanel()(evt);
-
-          trackEvent('quick_action', {
-            name: ACTIONS.INSERT_TEXT.trackingEventName,
-            element: 'none',
-          });
-        },
-        onMouseDown: handleMouseDown,
-      },
-    ];
-  }, [
-    actionMenuProps,
-    backgroundElement,
-    handleFocusMediaPanel,
-    handleFocusPageBackground,
-    handleFocusTextSetsPanel,
-    handleMouseDown,
-  ]);
+  const noElementSelectedActions = useMemo(
+    () => (
+      <>
+        <ContextMenuComponents.Button
+          onClick={(evt) => {
+            handleFocusPageBackground(backgroundElement?.id)(evt);
+
+            trackEvent('quick_action', {
+              name: ACTIONS.CHANGE_BACKGROUND_COLOR.trackingEventName,
+              element: 'none',
+            });
+          }}
+          {...actionMenuProps}
+        >
+          <ContextMenuComponents.Icon
+            title={ACTIONS.CHANGE_BACKGROUND_COLOR.text}
+          >
+            <Bucket />
+          </ContextMenuComponents.Icon>
```

Comment:
In my opinion hooks shouldn't return react elements. I think I'll revert this to what it was and used the returned arrays of objects to render the UI we want.